Rubberduck.RibbonDispatcher/Rubberduck.ExcelRibbon2013/ThisAddIn.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Xml.Linq;
+using Excel = Microsoft.Office.Interop.Excel;
+using Office = Microsoft.Office.Core;
+using Microsoft.Office.Tools.Excel;
+
+namespace Rubberduck.RibbonDispatcher.Excel2013 {
+    using Office = Microsoft.Office.Core;
+
+    public partial class ThisAddIn
+    {
+        private void ThisAddIn_Startup(object sender, System.EventArgs e)
+        {
+        }
+
+        private void ThisAddIn_Shutdown(object sender, System.EventArgs e)
+        {
+        }
+
+        RubberduckRibbon _rubberduckRibbon;
```

Comment:
members below methods?

Reply:
A lot of my answers are simply - I never thought about that. One or two points I would push back on, but as for the rest, just point me at the guidelines.

Reply:
@Vogel612 : Thank you for your time.  It is a tremendous pleasure to have my code read, and receive comments more profound than "Why are you defining all these classes?", the comment I got after a team presentation last spring.